src/traffic/core/flight.py

```diff
@@ -1658,6 +1658,7 @@ def resample(
         self,
         rule: str | int = "1s",
         how: None | str | dict[str, Iterable[str]] = "interpolate",
+        interpolation_options: dict[str, Any] = {},
```

Comment:
Since the parameter is `interpolate`, the usual convention is the remain of the code would make it `interpolate_kw`.

Reply:
Ok, makes sense, this way it is clear that the options belong to the pandas `interpolate` method. Is `_kw` also a convention you are following at the moment? For unfamiliar users it may be less obvious than `_kwargs` or `_options`. But I don't have a strong opinion on that, let me know which name you prefer and I will adapt the PR.

Reply:
True, it may be not that obvious, but if I change it here, I'd rather change it everywhere else as well, and then things break, etc.

Another reasonable option would be `.resample(how={'interpolate': {"method": "cubic"}})` but then it can become messy with the other (arguably shitty) design decision with `.resample(how={'interpolate': ['altitude', 'groundspeed']})`.

So... maybe stick with `interpolate_kw` for the moment, unless you have a magic idea to solve this all...

Reply:
Ok, then `interpolate_kw` is the best option. I renamed the argument now.